reference/src/unit-testing.md

```diff
@@ -2,20 +2,20 @@
 
 Unit testing for Move uses three annotations in the Move source language:
 
-- `#[test]`
-- `#[test_only]`, and
-- `#[expected_failure]`.
+- `#[test]`,
+- `#[expected_failure]`, and
+- `#[test_only]`
 
-They respectively mark a function as a test, mark a module or module member (`use`, function, or
-struct) as code to be included for testing only, and mark that a test is expected to fail. These
-annotations can be placed on a function with any visibility. Whenever a module or module member is
-annotated as `#[test_only]` or `#[test]`, it will not be included in the compiled bytecode unless it
-is compiled for testing.
+They respectively mark a function as a test, mark that a test is expected to fail, and mark a module
+or module member ([`use`](./uses.md), [function](./functions.md), [struct](./structs.md), or
+[constant](./constants.md)) as code to be included for testing only. These annotations can be placed
+on a function with any visibility. Whenever a module or module member is annotated as `#[test_only]`
+or `#[test]`, it will not be included in the compiled bytecode unless it is compiled for testing.
```

Comment:
Revision proposal:
```
- `#[test]` -- marks a function as a test;
- `#[expected_failure]` -- marks that a test is expected to fail;
- `#[test_only]` -- marks a module or module member ([`use`](./uses.md), [function](./functions.md),
  [struct](./structs.md), or [constant](./constants.md)) as code to be included for testing only.

These annotations can be placed on any appropriate form with any visibility. Whenever a module or
module member is annotated as `#[test_only]` or `#[test]`, it will not be included in the compiled
bytecode unless it is compiled for testing.
```